data_ingestion.ipynb

```diff
@@ -22,7 +22,7 @@
         "colab_type": "text"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    if SHARED_DRIVE:
</pre><p>For the export, we need a <code>LOCAL_DRIVE</code> option as discussed. We want contributors to be able to use their local drive so that we don't give a bunch of people write-access to sensitive folders while we work on enabling EarthEngine for medium-to-long-term governance for this.</p><p> </p><blockquote>maybe the first colab can also set everything up in a local drive -- including rasters.&nbsp;</blockquote><blockquote>and the downstream colabs can all work with local files.</blockquote><blockquote>this also means you can narrow the scope of the EE changes to only be in ingestion</blockquote><blockquote>including an EE raster to local raster</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Okay, so only the personal/local drive for the output. Not shared.

Reply:
Yep.